### PR TITLE
Huseyn watch airpods

### PR DESCRIPTION
**Pull request by Huseynisgenderov**
very good

### CODE REVIEW

**Comment by aliihusyn** [comment]
okay

File: watch.html [hunk 1]
```diff
@@ -108,21 +108,169 @@
...
-        >
-          Hüseyn İsgəndərov
-        </h2>
+      <section class="watch-bar">
+        <nav class="watch-navBar">
+          <div class="row-navBar">
+            <ul class="watch-items">
+              <li class="item">
+                <div class="figure">
+                  <a href="#">
```
Comment (alidjanoff) [inline watch.html:114]: Real sehifelerin linklerin elave et

File: assets/scss/_watch.scss [hunk 2]
```diff
@@ -24,6 +24,76 @@
...
+              height: 54px;
+            }
+          }
+          &:nth-child(13) {
+            padding: 0 4px 0 9px;
+            .figure {
+              width: 63px;
+              height: 54px;
+            }
+          }
```
Comment (alidjanoff) [inline assets/scss/_watch.scss:96]: Like eledim

File: watch.html [hunk 3]
```diff
@@ -272,9 +269,33 @@
...
+      <section class="watch-ultra-section">
+        <div class="watch-colum">
+          <div class="watch-ultra-content">
+            <div class="content-size">
+              <img
+                src="./assets/images/watch-airpods/logo_watch_ultra.png"
+                alt="watch-ultra"
+              />
+              <p class="headline">Adventure awaits.</p>
+              <p class="price">$799</p>
```
Comment (alidjanoff) [inline watch.html:283]: Dollar iwaresi ayri, qiymet ise span in icinde olsun

**Pull request merged**